app/javascript/components/NewService.js

```diff
@@ -42,7 +42,9 @@ class NewService extends React.Component {
     const { errorMessages } = this.state;
     return (
       <div>
-        <ServiceForm onSubmit={this.createService} errorMessages={errorMessages} />
+        <ServiceForm onSubmit={this.createService} errorMessages={errorMessages}>
+          登録
+        </ServiceForm>
```

Comment:
[IMO] Form で挟まれている文字列が Button のための文字列というのが読み取りづらいインターフェースになっちゃってるかもですね〜 😁
children よりも props で渡した方がわかりやすいインターフェースになると思います〜😃 

e.g. 
```suggestion
        <ServiceForm
           onSubmit={this.createService}
           errorMessages={errorMessages}
           action="create"
         />
```

Reply:
ありがとうございます!
確かにその方がわかりやすいように思いました。
propsで渡すように修正しました。